src/Views/Components/ListTable/ListTablePage/index.tsx

```diff
@@ -156,8 +166,18 @@ export default function ListTablePage({
         setSelectedIds(selected);
         setSelectedNames(names);
         if (selected.length) {
-            setModalContent({...bulkActions[actionIndex]});
-            dialog.current.show();
+            if ('urlAction' === bulkActions[actionIndex].type) {
```

Comment:
I just moved the current logic to the `confirm` function that already exists in the `BulkActionsConfig` type. So, the only difference related to the `urlAction` is that it will trigger the `confirm` function directly instead of loading their content inside the generic modal used by the list table component. Resolved in 5223c88c002220cb4fd244d1ec5f6c951a7cc233

Reply:
Yeah, it looks much better this way. Thanks!